src/cogent3/core/profile.py

```diff
@@ -292,6 +293,53 @@ def entropy(self):
         result = self.entropy_terms().row_sum()
         return result.array
 
+    def relative_entropy_terms(self, background=None):
+        """
+        Parameters
+        ----------
+        background : dict
+            {motif_1: prob_1, motif_2: prob_2, ...} is the specified background distribution.
+
+        Returns
+        -------
+        self.template.wrap(ret) : DictArray
```

Comment:
just state it returns a DictArray of position-wise relative entropy terms per motif

Reply:
you still did not address this comment